asterisk_mbox_server/cdr.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, cdr_path, sleep=5.0):
                 if sys.platform.startswith('linux'):
                     import inotify.adapters
                     self._inot = inotify.adapters.Inotify()
-                    self._inot.add_watch(self._cdr_file.encode('utf-8'))
+                    self._inot.add_watch(self._cdr_file)
```

Comment:
This was specifically added because it was causing issues.  I think there may be 2 incompatible versions in play. need a solution that works everywhere

Reply:
I will check this out... For me it works without the encode.